docs/demos-examples/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: demos-examples
+permalink: /demos-examples/
+
+# Banner Content
+# =================================
+title: Demos and Examples
+subtitle:
+hero_image: /assets/img/home-callouts/demos_examples.jpg
+
+# Demos
+# =================================
+demos:
+ - title: Digital Slide Archive
+   description: "The data from the collaborative annotation study is available on a demo instance of the Digital Slide Archive. The link goes to one of the TCGA slides that was used in the study. If you click the"
+   description2: "image icon in the Annotations panel on the right side of the screen, you'll see the results of a collaborative annotation."
+   link: http://demo.kitware.com/histomicstk/histomicstk#?image=5bbdee4fe629140048d01adf&bounds=25823%2C19101%2C43534%2C29208%2C0
+   link_text: Demo
+---
```

Comment:
Can we add a screenshot to this page?  Attached is a possible screenshot, though the bottom edge needs trimming:
![screencapture-demo-kitware-histomicstk-histomicstk-2020-04-02-08_50_51](https://user-images.githubusercontent.com/8781639/78252158-65e0de00-74c0-11ea-830f-e59ad398cd50.png)

Reply:
Fixed in 58cff8b3c430c465c927273565e363442fde86ae